docs/conf.py

```diff
@@ -31,7 +31,6 @@
 copyright = "2023, TrueLearn"
 author = "TrueLearn Team"
 
-# pylint: disable=wrong-import-position
```

Comment:
i see you removed this

Reply:
Because you won't get an error from prospector because we specify in `prospector.yaml` to ignore `docs`.